From 3f9c2fd648eb7804f62e272b423916dd62e99a9d Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Sun, 30 Sep 2018 18:35:46 -0700
Subject: [PATCH 4/7] DO-NOT-UPSTREAM: Disable GCOV_KERNEL for Clang

Patches currently in testing, just don't want to add them to my stack.

Link: https://github.com/ClangBuiltLinux/linux/issues/17
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 kernel/gcov/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/gcov/Kconfig b/kernel/gcov/Kconfig
index 1e3823fa799b..8d3dc2303618 100644
--- a/kernel/gcov/Kconfig
+++ b/kernel/gcov/Kconfig
@@ -2,7 +2,7 @@ menu "GCOV-based kernel profiling"
 
 config GCOV_KERNEL
 	bool "Enable gcov-based kernel profiling"
-	depends on DEBUG_FS
+	depends on DEBUG_FS && !CC_IS_CLANG
 	select CONSTRUCTORS if !UML
 	default n
 	---help---
-- 
2.20.0

